tsconfig.declarations.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "noEmit": false,
+    "emitDeclarationOnly": true,
+    "outDir": "./declarations"
+  },
+  "include": [
+    "declarations.d.ts",
+    "declarations/*.d.ts",
+    "lib/**/*.js"
```

Comment:
I think all files is unnecessary, plugins and webpack.js (as Node.js API) maybe.. don't know exactly

Reply:
You are totally right. We just need to know what types we want to export.